main.go

```diff
@@ -43,20 +46,43 @@ func simpleTask(task Task, _, _ string) error {
 	return run(getContext(task.name), nil, task.cmd[0], task.cmd[1:]...)
 }
 
-var buildCMD = []string{"-mod=vendor", "-o", "app", "main.go"}
-var runCMD = []string{"./app"}
+var buildCMD = "-mod=vendor -o app main.go"
+var runCMD = "./app"
 
 func main() {
-	if len(os.Args) > 1 {
-		buildCMD = strings.Split(os.Args[1], " ")
-		if len(os.Args) > 2 {
-			runCMD = strings.Split(os.Args[2], " ")
+	yamlFile, err := ioutil.ReadFile("wtc.yaml")
+	if err != nil {
+		if len(os.Args) > 1 {
+			buildCMD = os.Args[1]
+			if len(os.Args) > 2 {
+				runCMD = os.Args[2]
+			}
+		}
+		config = configuration.Config{
+			Build: buildCMD,
+			Run:   runCMD,
+			Rules: []configuration.Rule{
+				configuration.Rule{
+					Name:     "Test",
+					Regex:    `_test\.go$`,
+					Ignore:   "/vendor/",
+					Debounce: 500,
+					Command:  "test -mod=vendor -cover",
+				},
```

Comment:
You should remove Tasks below and use this config.Rules instead.
For default, you can add two rules; "Test" and "Go".